source/material_model/interface.cc

```diff
@@ -240,6 +240,44 @@ namespace aspect
       cell (NULL)
     {}
 
+    template <int dim>
+    MaterialModelInputs<dim>::MaterialModelInputs(const DataPostprocessorInputs::Vector<dim> &input_data,
+                                                  const Introspection<dim> &introspection,
+                                                  const bool use_strain_rate)
+      :
+      position(input_data.solution_values.size()),
+      temperature(input_data.solution_values.size(), numbers::signaling_nan<double>()),
+      pressure(input_data.solution_values.size(), numbers::signaling_nan<double>()),
+      pressure_gradient(input_data.solution_values.size(), numbers::signaling_nan<Tensor<1,dim> >()),
+      velocity(input_data.solution_values.size(), numbers::signaling_nan<Tensor<1,dim> >()),
+      composition(input_data.solution_values.size(), std::vector<double>(introspection.n_compositional_fields, numbers::signaling_nan<double>())),
+      strain_rate(input_data.solution_values.size(), numbers::signaling_nan<SymmetricTensor<2,dim> >()),
+      cell(NULL)
+    {
+      this->position = input_data.evaluation_points;
```

Comment:
you can actually do this assignment already in the initialization above (first line of member initialization)